lms/djangoapps/teams/static/teams/js/spec/edit_team_spec.js

```diff
@@ -0,0 +1,189 @@
+            teamEditView = new TeamEditView({
+                el: $('.teams-content'),
+                teamParams: {
+                    teamsUrl: teamsUrl,
+                    courseId: "a/b/c",
+                    topicId: 'awesomeness',
+                    topicName: 'Awesomeness',
+                    languages: [['a', 'aaa'], ['b', 'bbb']],
+                    countries: [['c', 'ccc'], ['d', 'ddd']]
+                }
+            }).render();
+        });
+
+        it('can render itself correctly', function () {
+            expectContent(
+                '.team-instructions',
+                "Enter information to describe your team. You cannot change these details after you create the team."
+            );
+
+            expectContent('.u-field-name .u-field-title', "Team Name (Required) *");
+            expectContent(
+                '.u-field-name .u-field-message-help', "A name that identifies your team (maximum 255 characters)."
+            );
+            expectContent('.u-field-description .u-field-title', "Team Description (Required) *");
+            var descriptionFieldMessage = "A short description of the team to help other learners understand " +
+                "the goals or direction of the team (maximum 300 characters).";
+            expectContent('.u-field-description-message', descriptionFieldMessage);
+            expectContent('.u-field-optional_description .u-field-title', "Optional Characteristics");
+            var optionalDescriptionMessage = "Help other learners decide whether to join your team by specifying " +
+                "some characteristics for your team. Choose carefully, because fewer people might be interested in " +
+                "joining your team if it seems too restrictive.";
+            expectContent('.u-field-optional_description .u-field-message-help', optionalDescriptionMessage);
+            expectContent('.u-field-language .u-field-title', "Language");
+            expectContent(
+                '.u-field-language .u-field-message-help',
+                "The language that team members primarily use to communicate with each other."
+            );
+            expectContent('.u-field-country .u-field-title', "Country");
+            expectContent(
+                '.u-field-country .u-field-message-help', "The country that team members primarily identify with."
+            );
+
+            verifyDropdownData('.u-field-language select', [['a', 'aaa'], ['b', 'bbb']]);
+            verifyDropdownData('.u-field-country select', [['c', 'ccc'], ['d', 'ddd']]);
+
+            expect(teamEditView.$('.create-team.form-actions .action-primary').length).toBe(1);
+            expect(teamEditView.$('.create-team.form-actions .action-cancel').length).toBe(1);
+        });
+
+        it('can create a team', function () {
+            var requests = AjaxHelpers.requests(this);
+
+            teamEditView.$('.u-field-name input').val('TeamName');
```

Comment:
Could you please use `teamsData.data` and `teamsData.description` here (and on lines 172-173)?